roles/taca/defaults/main.yml

```diff
@@ -9,7 +9,7 @@ flowcell_parser_version: cfe18abb6b758a6389a9d6aa75f1866d517dc36e
 
 taca_repo: https://github.com/SciLifeLab/TACA.git
```

Comment:
Should be https://github.com/NationalGenomicsInfrastructure/TACA.git, no?

Reply:
Good catch!